podcasts/Sharing/SharingView.swift

```diff
@@ -37,13 +39,39 @@ struct SharingView: View {
         TabView(selection: $selectedMedia) {
             ForEach(ShareImageStyle.allCases, id: \.self) { style in
                 ShareImageView(info: selectedOption.imageInfo, style: style)
+                    .clipShape(RoundedRectangle(cornerRadius: 12))
                     .tabItem { Text(style.tabString) }
             }
         }
         .tabViewStyle(.page)
     }
+
+    @ViewBuilder var buttons: some View {
+        HStack(spacing: 24) {
+            ForEach(destinations, id: \.self) { option in
+                Button {
+                    option.action(selectedOption, selectedMedia)
+                } label: {
+                    VStack {
+                        option.icon
+                            .renderingMode(.template)
+                            .font(size: 20, style: .body, weight: .bold)
+                            .frame(width: 24, height: 24)
+                            .padding(15)
+                            .background {
+                                Circle()
+                                    .foregroundStyle(.white.opacity(0.1))
+                            }
+                        Text(option.name)
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                    }
+                }
+            }
+        }
+    }
 }
 
 #Preview {
-    SharingView(selectedOption: .podcast(Podcast.previewPodcast()))
+    SharingView(destinations: [.copyLinkOption], selectedOption: .podcast(Podcast.previewPodcast()))
```

Comment:
Maybe do something like this so we can actually see the buttons in the preview?

```suggestion
    #Preview {
    ZStack {
        Color.black

        SharingView(destinations: [.copyLinkOption], selectedOption: .podcast(Podcast.previewPodcast()))
    }
}

```